File: Algorithms&ProblemSolving/BreakingTheRecords.py
import math
import os
import random
import re
import sys

#scores = [3 ,4 ,21 ,36 ,10 ,28 ,35 ,5 ,24 ,42]
def breakingRecords(scores):
    lowest = scores[0]
    highest = scores[0]
    lowestCount = 0
    highestCount = 0
    x = 0
    while x < len(scores):
        if scores[x] < lowest:
            lowest = scores[x]
            lowestCount += 1
        if scores[x] > highest:
            highest = scores[x]
            highestCount += 1
        x += 1
    print(highestCount,lowestCount)

File: Algorithms&ProblemSolving/BreakingTheRecords.java
public class BreakingTheRecords{

static int[] breakingRecords(int[] scores) {
    int lowest = scores[0];
    int highest = scores[0];
    int lowestCount = 0;
    int highestCount = 0;
    
    for (int i = 0; i < scores.length; i++){
        if (scores[i] < lowest){
            lowest = scores[i];
            lowestCount++;
        }
        if (scores[i] > highest){
            highest = scores[i];
            highestCount++;
        }
    }
    int[] res = new int[2];
    res[0] = highestCount;
    res[1] = lowestCount;
    return res;
}
}